src/icesat2_tracks/analysis_db/B02_make_spectra_gFT.py

```diff
@@ -1,32 +1,40 @@
+import sys
+
+
 """
 This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
 This is python 3
 """
```

Comment:
In a comment in a previous PR John pointed out that it's better to put module docstrings at the top. They serve as module metadata. For example, `help(mymodule)` will print such a docstring. Note that this was the case in the previous commit.

```suggestion
"""
This file open a ICEsat2 track applied filters and corections and returns smoothed photon heights on a regular grid in an .nc file.
This is python 3
"""

import sys
```